justfile

```diff
@@ -0,0 +1,10 @@
+configure cfg="Debug" dir="build":
+    cmake -B {{dir}} -S . -DUSE_VCPKG=ON -DBUILD_TESTS=ON -DBUILD_BENCHMARKS=ON -DCMAKE_EXPORT_COMPILE_COMMANDS=1 -DCMAKE_BUILD_TYPE={{cfg}}
+
+build dir="build":
+    cmake --build {{dir}}
+
+test cfg="Debug" dir="build":
+    @just configure {{cfg}} {{dir}}
+    @just build {{dir}}
+    {{dir}}/tests/cachemere_tests
```

Comment:
thought: There's something funny in seing a Make-like program being used to call CMake which is itself a build system generator that (can) generates Make files to build the program. :laughing: It feels like

![image](https://user-images.githubusercontent.com/8481892/218523354-066d5f8b-531f-4a37-8213-6eb38759e08e.png)